README.md

```diff
@@ -14,7 +14,7 @@ This repository generates the build-products for both backends in `.build/defn/j
 
 ### System Dependencies
 
-The following are needed for building/running KEVM:
+The following are needed for building/running KWASM:
```

Comment:
We should maybe start opting for "KWasm" or similar. "Wasm" isn't an acronym and therefore not capitalized, so the official abbreviation is "Wasm".

Reply:
KWASM => KWasm, and WASM => WebAssembly throughout the repository.

This debate has come up from time to time on the ewasm team, it seems that WebAssembly is the only non-controversial choice. For KWasm I figure we can pick whatever name we like, but probably will be less controversial with KWasm.